code/_debugger.dm

```diff
@@ -2,7 +2,6 @@
 //Datum gets created in master.dm because for whatever reason global code in there gets runs first
 //In case we ever figure out how to manipulate global init order please move the datum creation into this file
 /datum/debugger
```

Comment:
to be completely honest I do not know what the debugger does and had to remove it from world.dm to stop everything from hanging on startup